### DISCUSSION TITLE
Is it possible to support Dify?

### DISCUSSION THREAD

**Discussion by jtsang4** [langfuse/langfuse#1744] [Ideas]
### Describe the feature or potential improvement

[Dify](https://github.com/langgenius/dify) is an open-source LLM app development platform, it is popular and fast growing( see: https://ossinsight.io/collections/llm-tools/) and it also supports workflow like langflow, flowise recently, so I'm wondering is it possible that langfuse could supports Dify.

**Comment by clemra** [comment]
thank you @jtsang4 -- dify looks nice and would fit our roster next to [langchain](https://langfuse.com/docs/integrations/langchain/tracing) and [llama index](https://langfuse.com/docs/integrations/llama-index/get-started). 

We'll have a look but also open for contributions.

+ adding #1221 for visibility here.

**Comment by clemra** [comment]
Dify integration is now live!
https://x.com/langfuse/status/1806779179859591640
https://langfuse.com/docs/integrations/dify

**Reply by clemra** [reply]
@jtsang4
